internal/cmd/beta/network-interface/list/list.go

```diff
@@ -83,6 +83,9 @@ func NewCmd(p *print.Printer) *cobra.Command {
 					p.Debug(print.ErrorLevel, "get network name: %v", err)
 					networkLabel = *model.NetworkId
 				}
+				if networkLabel == "" {
```

Comment:
```suggestion
				else if networkLabel == "" {
```

can be an else if, the same value was already assigned inside the previous if statement

Reply:
yes makes sense. I'll update it